### ISSUE TITLE
Fails to attach to server after using docker compose up command

### ISSUE THREAD

**Issue by delusionalfox** [heartexlabs/label-studio#4785]
Attempting to install ml backend using docker instructions. Upon trying to start the Ml backend with docker compose up command it returns:
[+] Running 1/0
 ✔ Container server  Created                                                                                       0.0s
Attaching to server
server  | exec /app/start.sh: no such file or directory
server exited with code 1

Potentially relatedly when running the freshly installed label studio image with docker it returns this error, however, otherwise starts normally:
Can't read version file: ls-version_.py. Fall back to: version_.py
[2023-09-15 06:53:31,675] [root::_read_py::61] [WARNING] Can't read version file: ls-version_.py. Fall back to: version_.py
[2023-09-15 06:53:32,358] [root::_read_py::61] [WARNING] Can't read version file: ls-version_.py. Fall back to: version_.py
Performing system checks...

System check identified no issues (1 silenced).

**Comment by jonathan-attard** [comment]
Hi, I have the same issue, any luck solving this?

**Comment by delusionalfox** [comment]
No, I haven't. Tried changing the code to avoid cashe but still no dice

**Comment by DomPka** [comment]
I have the same problem. I've installed everything using powershell maybe that's the problem.
